app/src/main/java/de/hbch/traewelling/api/models/user/User.kt

```diff
@@ -17,5 +17,6 @@ data class User(
     @SerializedName("privateProfile") val privateProfile: Boolean,
     @SerializedName("home") var home: Station?,
     @SerializedName("prevent_index") val preventIndex: Boolean,
-    @SerializedName("language") val language: String
+    @SerializedName("language") val language: String,
+    @SerializedName("default_status_visibility") val defaultStatusVisibility: StatusVisibility
```

Comment:
```suggestion
    @SerializedName("defaultStatusVisibility") val defaultStatusVisibility: StatusVisibility
```

You changed this name in the other PR :)